src/Compilers/CSharp/Portable/Symbols/LabelSymbol.cs

```diff
@@ -13,24 +13,6 @@ namespace Microsoft.CodeAnalysis.CSharp.Symbols
     /// </summary>
     internal abstract class LabelSymbol : Symbol, ILabelSymbol
     {
-        internal LabelSymbol(string name)
```

Comment:
moved to derived classes where it can be made lazy. We rarely need the name and computing/allocating it is relatively expensive.